code/API_code/CAMARA_numberVerify.postman_collection.json

```diff
@@ -0,0 +1,444 @@
+{
```

Comment:
Why adding the Postman collection ? is it something that should be provided for all Camara API as I'm not sure there is this for other API proposal.